hw/top_earlgrey/data/ip/chip_aon_timer_testplan.hjson

```diff
@@ -0,0 +1,125 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_aon_timer
+  testpoints: [
+    {
+      name: chip_sw_aon_timer_wakeup_irq
+      desc: '''Verify the AON timer wake up interrupt in normal operating state.
+
+            - Program the PLIC to let the AON timer wake up interrupt the CPU.
+            - Program the AON timer to generate the wake up timeout interrupt after some time.
+            - Issue a WFI to wait for the interrupt to trigger.
+            - Service the interrupt when it triggers; verify that it came from AON timer.
+            - Verify that the interrupt triggered only after the timeout elapsed.
+            '''
```

Comment:
Add any SiVal specifc comments, if there are some.

Reply:
I think if the tests can run in cw310 it is a good idea to make sure they work okay, and to add the corresponding cw310 test in the bazel field: this makes it clear all that is needed is replacing cw310 by silicon. If the test can run in cw310 but fails I would file an issue. This will make it easier to estimate the work remaining.

Reply:
done